client/lib/plans/features-list.js

```diff
@@ -1338,6 +1349,47 @@ export const FEATURES_LIST = {
 		getSlug: () => FEATURE_JETPACK_VIDEOPRESS_STORAGE,
 		getTitle: () => i18n.translate( '1TB of storage' ),
 	},
+
+	/* START - Jetpack tiered product-specific features */
+	[ FEATURE_JETPACK_10GB_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_JETPACK_10GB_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( '10GB of backup storage' ),
+	},
+	[ FEATURE_JETPACK_1TB_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_JETPACK_1TB_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( '1TB of backup storage' ),
+	},
+	[ FEATURE_JETPACK_30_DAY_ARCHIVE_ACTIVITY_LOG ]: {
+		getSlug: () => FEATURE_JETPACK_30_DAY_ARCHIVE_ACTIVITY_LOG,
+		getTitle: () => i18n.translate( '30-day archive & activity log' ),
+	},
+	[ FEATURE_JETPACK_1_YEAR_ARCHIVE_ACTIVITY_LOG ]: {
+		getSlug: () => FEATURE_JETPACK_1_YEAR_ARCHIVE_ACTIVITY_LOG,
+		getTitle: () => i18n.translate( '1-year archive & activity log' ),
+	},
+	[ FEATURE_JETPACK_PRODUCT_BACKUP ]: {
+		getSlug: () => FEATURE_JETPACK_PRODUCT_BACKUP,
+		getTitle: () => i18n.translate( 'All Backup features' ),
+		isHighlighted: () => true,
+	},
+	[ FEATURE_JETPACK_PRODUCT_VIDEOPRESS ]: {
+		getSlug: () => FEATURE_JETPACK_PRODUCT_VIDEOPRESS,
+		getTitle: () => i18n.translate( 'VideoPress' ),
+	},
+	[ FEATURE_JETPACK_ALL_BACKUP_SECURITY_FEATURES ]: {
+		getSlug: () => FEATURE_JETPACK_ALL_BACKUP_SECURITY_FEATURES,
+		getTitle: () => i18n.translate( 'All Backup & Security features' ),
+	},
+	[ FEATURE_JETPACK_REAL_TIME_CLOUD_BACKUPS ]: {
+		getSlug: () => FEATURE_JETPACK_REAL_TIME_CLOUD_BACKUPS,
+		getTitle: () => i18n.translate( 'Real-time cloud backups' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Real-time backups' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/tiered-product-cards-copy)